deployment-configuration/compose/templates/allvalues-template.yaml

```diff
@@ -0,0 +1,20 @@
+{{- /*
```

Comment:
Not sure what's the purpose of this template. I'm seeing two files generated -- allvalues.yaml and  allvalue2.yaml, with the same content as values.yaml

Reply:
It's a forgotten experiment that is here. I saw one template for the k8s target that was copying/reproducing the `allvalues.yaml` files, but I wasn't sure if it was replicating it exactly as the original file or if it was modifying entries at the same time. I understood later that the logic for this template is actually to have the content of the `allvalues.yaml` inside of the k8s cluster, while the approach of docker compose is different as we can mount the file in the container. I need to remove this generation, thanks for spotting it!